rules/Transform/Rector/Class_/RemoveAllowDynamicPropertiesAttributeRector.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Transform\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Php80\NodeAnalyzer\PhpAttributeAnalyzer;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @changelog https://wiki.php.net/rfc/deprecate_dynamic_properties
+ *
+ * @see \Rector\Tests\Transform\Rector\Class_\RemoveAllowDynamicPropertiesAttributeRector\RemoveAllowDynamicPropertiesAttributeRectorTest
+ */
+final class RemoveAllowDynamicPropertiesAttributeRector extends AbstractRector implements ConfigurableRectorInterface
```

Comment:
please rebase and change to implements `\Rector\Core\Contract\Rector\AllowEmptyConfigurableRectorInterface` so it won't show warning when not configured.